sources/packages/backend/apps/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -141,3 +142,12 @@ export class SearchInstitutionAPIOutDTO {
   operatingName: string;
   address: AddressAPIOutDTO;
 }
+
+export class SearchInstitutionQueryAPIInDTO {
+  @IsOptional()
+  @MaxLength(64)
```

Comment:
I guess that these numbers (lines 148 and 151) came from the DB restrictions. We can have those ones centralized in the entity model for these cases in the same way we have `NOTES_MAX_CHARACTERS`.
If not the case we can have those centralized is some constant file.